### DISCUSSION TITLE
Error: Got unexpected extra argument for --user

### DISCUSSION THREAD

**Discussion by bthahn** [Serene-Arc/bulk-downloader-for-reddit#553] [Q&A] [answered]
When I run the following in powershell I get  - Error: Got unexpected extra argument ( --user user1 --user user2 --user user3 --user user4 --user user5 --user user6 --user user7 --user user8)

`
$userlist= "--user user1 --user user2 --user user3 --user user4 --user user5 --user user6 --user user7 --user user8"

python -m bdfr download $OutputPath " $userlist" --submitted -v --no-dupes --search-existing --folder-scheme '{REDDITOR}' --file-scheme '{TITLE}_{POSTID}`

If I replace $userlist with  --user user1 --user user2 --user user3 --user user4 --user user5 --user user6 --user user7 --user user8 the command works correctly.  How do I  get it to use the content of the variable?

**Comment by Serene-Arc** [comment, marked as answer]
At a guess, it's because you have the quotes around $userlist in the command. This will input it as a single string and prevent splitting. You want the splitting, so it's causing the error.

**Reply by bthahn** [reply]
python -m bdfr download $OutputPath  $userlist --submitted -v --no-dupes --search-existing --folder-scheme '{REDDITOR}' --file-scheme '{TITLE}_{POSTID}'  

Gives me the error Error: No such option: --user user1 --user user2 --user user3 --user user4 --user user5 --user user6 --user user7 --user user8

**Reply by Serene-Arc** [reply]
I don't know Powershell sorry, look up variable splitting and see what the way to do it is. The BDFR looks like it's reading the entire string as a single option

**Reply by bthahn** [reply]
That was it! Thank you. 

running 

$userlist = $userlist -split " "

before the python command resolved the issue.

**Reply by Serene-Arc** [reply]
Great :) glad it worked
